### PR TITLE
done with the input component

### PR DESCRIPTION
**Pull request by ajit-sharmaGH**
-standard input
-textBox input
-validation check input

### CODE REVIEW

**Review by Raghav888 — APPROVED** [review]
Approved

File: CSS/image.css [hunk 1]
```diff
@@ -1,3 +1,68 @@
 @import url("base/reset.css");
 @import url("base/color.css");
-@import url("base/nav.css");
\ No newline at end of file
+@import url("base/nav.css");
+.image-body {
+    display: flex;
+    flex-direction: column;
```
Comment (Mohasin-Haque) [inline CSS/image.css:6]: You can make a class for flex so that you need not to use it again and again

File: CSS/input.css [hunk 2]
```diff
@@ -1,3 +1,58 @@
...
+
+.label-text {
+    display: flex;
+    flex-direction: column;
+    color: var(--dark);
+    margin: 2px;
+}
+
+.validation-check {
+    color: var(--green);
```
Comment (Mohasin-Haque) [inline CSS/input.css:40]: Try giving relative name.  If we have to use specific name then why not we use green directly.. Tjink

File: Htmlfiles/image.html [hunk 3]
```diff
@@ -71,26 +68,55 @@ <h4>Tungsten UI</h4>
...
+            <div class="iframe-parent_class">
+                <code class="iframe-child_class">
+                        <iframe frameborder="0" src="https://carbon.now.sh/embed?bg=rgba%2821%2C23%2C24%2C1%29&t=seti&wt=none&l=htmlmixed&width=677.5&ds=false&dsyoff=20px&dsblur=68px&wc=true&wa=true&pv=0px&ph=0px&ln=false&fl=1&fm=Hack&fs=12.5px&lh=152%25&si=false&es=2x&wm=false&code=%253Cheader%2520class%253D%2522sub-headings%2522%253ERound%2520Images%253C%252Fheader%253E%250A%2520%2520%2520%2520%2520%2520%2520%2520%253Cdiv%2520class%253D%2522image-container%2522%253E%250A%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%253Cimg%2520class%253D%2522img-sm-round%2520round-img%2520%2520images%2522%2520%252F%253E%250A%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%253Cimg%2520class%253D%2522img-lg-round%2520round-img%2520images%2522%2520%252F%253E%250A%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%253Cimg%2520class%253D%2522img-lgr-round%2520round-img%2520images%2522%2520%252F%253E%250A%2520%2520%2520%2520%2520%2520%2520%2520%253C%252Fdiv%253E" sandbox="allow-scripts allow-same-origin"></iframe>
```
Comment (Mohasin-Haque) [inline Htmlfiles/image.html:104]: Needs formating.. Please run prettier on this

**Review by Mohasin-Haque — COMMENTED** [review]
Your code looks good to me but needs to work on the comments.. You are good to merge then😊

File: CSS/image.css (hunk 1 above)
Comment (ajit-sharmaGH) [reply CSS/image.css:6]: okk thanks will do that

**Pull request merged**